From d4d10e417b23eed7e7432a68fc7490a981ae1a73 Mon Sep 17 00:00:00 2001
From: Xu Han <xu.han@intel.com>
Date: Wed, 30 Dec 2015 20:09:44 +0800
Subject: [PATCH 298/403] Add missing register "VEBOX_HWS_PGA_GEN7" during
 rebase.

* Add missing mmio register "VEBOX_HWS_PGA_GEN7", when mmio registers save/restore.
  This register is missed during rebase.
* Fix bug 855.

Signed-off-by: Xu Han <xu.han@intel.com>
---
 drivers/gpu/drm/i915/vgt/legacy/render_hsw.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/i915/vgt/legacy/render_hsw.c b/drivers/gpu/drm/i915/vgt/legacy/render_hsw.c
index c26a128..604954d 100644
--- a/drivers/gpu/drm/i915/vgt/legacy/render_hsw.c
+++ b/drivers/gpu/drm/i915/vgt/legacy/render_hsw.c
@@ -37,6 +37,7 @@ vgt_reg_t vgt_gen7_render_regs[] = {
 	BLT_HWS_PGA_GEN7,
 	RENDER_HWS_PGA_GEN7,
 	BSD_HWS_PGA_GEN7,
+	VEBOX_HWS_PGA_GEN7,
 	_REG_VECS_HWS_PGA,
 
 	_REG_BCS_MI_MODE,
-- 
1.7.10.4

